packages/react-core/src/scalprum-provider.tsx

```diff
@@ -11,46 +15,41 @@ export interface ScalprumState<T extends Record<string, any> = Record<string, an
 }
 
 export interface ScalprumProviderProps<T extends Record<string, any> = Record<string, any>> {
-  config: ScalprumFeed;
+  config: AppsConfig;
   api?: T;
   children?: React.ReactNode;
+  pluginSDKOptions?: {
+    pluginStoreFeatureFlags?: FeatureFlags;
+    pluginLoaderOptions?: Partial<PluginLoaderOptions>;
+    pluginStoreOptions?: Partial<PluginStoreOptions>;
```

Comment:
FYI, option types for `PluginStore` and `PluginLoader` are already using `Partial<T>`.